libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.kt

```diff
@@ -26,23 +27,38 @@ import retrofit2.Callback
  * [MapboxNavigation.startNavigation], using this class will ensure your
  * request includes all the proper information needed for the navigation session to begin.
  *
- *
- *
- *
  * Developer Note: MapboxDirections cannot be directly extended since it is an AutoValue class.
  *
  * 0.5.0
  */
-private val EVENT_LISTENER = NavigationRouteEventListener()
 
 class NavigationRoute
-/**
- * Package private constructor used for the [Builder.build] method.
- *
- * @param mapboxDirections a new instance of a [MapboxDirections] class
- * @since 0.5.0
- */
-internal constructor(private val mapboxDirections: MapboxDirections) {
+internal constructor(
+    private val mapboxDirections: MapboxDirections
```

Comment:
It's pretty simple to add new line. Lets back it to single

Reply:
imo it looks better, like 
```
class ClassA 
   @JvmOverloads 
   constructor(
   val a: String, 
   val veryVeryLongNameAAAAAAAAAAAA,
   ...
```
better than this 
```
class ClassA @JvmOverloads constructor(val a: String, val veryVeryLongNameAAAAAAAAAAAA, ...)
```

Reply:
Yeah, but here we have only 1 param, without @JvmOverloads, without parent classes and so on.... But yeah, it just NIT, so, as you wish, man)